src/components/portal/portal.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
-import { Portal as ReactPortal }  from 'react-portal';
 import PropTypes from 'prop-types';
+import { createPortal } from 'react-dom';
 
+const isDomAvailable = !!(
```

Comment:
This would be a useful function to belong on our browser helper https://github.com/Sage/carbon/blob/master/src/utils/helpers/browser/browser.js